code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -348,6 +348,94 @@
 	else
 		STOP_PROCESSING(SSobj, src)
 
+//junglefowl
+/mob/living/simple_animal/hostile/retaliate/chicken
+	name = "\improper junglefowl"
+	desc = "A small, fierce, feathered beast."
+	gender = FEMALE
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	icon_state = "chicken_brown"
+	icon_living = "chicken_brown"
+	icon_dead = "chicken_brown_dead"
+	speak = list("Cluck!","BWAAAAARK BWAK BWAK BWAK!","Bwaak bwak.")
+	speak_emote = list("clucks","croons")
+	emote_hear = list("clucks.")
+	emote_see = list("pecks at the ground.","flaps its wings viciously.")
+	density = FALSE
+	speak_chance = 2
+	turns_per_move = 3
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab/chicken = 2)
+	var/egg_type = /obj/item/reagent_containers/food/snacks/egg
+	food_type = list(/obj/item/reagent_containers/food/snacks/grown/wheat)
+	response_help_continuous = "pets"
+	response_help_simple = "pet"
+	response_disarm_continuous = "gently pushes aside"
+	response_disarm_simple = "gently push aside"
+	response_harm_continuous = "scratches"
+	response_harm_simple = "scratch"
+	attack_verb_continuous = "scratches"
+	attack_verb_simple = "scratch"
+	health = 25
+	maxHealth = 25
+	var/eggsleft = 0
+	var/eggsFertile = TRUE
+	var/body_color
+	var/icon_prefix = "chicken"
+	pass_flags = PASSTABLE
+	mob_size = MOB_SIZE_SMALL
+	var/list/feedMessages = list("It clucks happily.","It clucks happily.")
+	var/list/layMessage = EGG_LAYING_MESSAGES
+	var/list/validColors = list("brown","black","white")
+	gold_core_spawnable = FRIENDLY_SPAWN
+	var/static/chicken_count = 0
+	environment_smash = ENVIRONMENT_SMASH_NONE
+	melee_damage_lower = 3
+	melee_damage_upper = 10
+
+	footstep_type = FOOTSTEP_MOB_CLAW
+
+/mob/living/simple_animal/chicken/hostile/retaliate/Initialize()
```

Comment:
the order here is wrong; presumably this has been copypasted from the /mob/living/simple_animal/chicken?

Reply:
correct, let me fix that